ansible/roles/rems-watcher/templates/restart_rems.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+FILE=/var/log/messages
+LINEFILE=/tmp/rems_error_track/line-track.txt
+myPath=/tmp/rems_error_track
+
+# This script restarts rems if it is not running properly
+# Check that the messages file exists
+# If the messages file exists but no line-track.txt file exists, create the file with the line number of the latest error msg and length of the messages file
+if [ -r "$FILE" ] && [ ! -f /tmp/rems_error_track/line-track.txt ]
+then
+        latestError=$(grep -n -P '^(?=.*org.apache.lucene.index.IndexNotFoundException)(?=.*no segments)' "$FILE" | tail -n1 | cut -d: -f1)
```

Comment:
Store this regexp `'^(?=.*org.apache.lucene.index.IndexNotFoundException)(?=.*no segments)'` in variable and use it?